<launch>
  
  <node name="roloc_graph_node" pkg="roloc_2d" type="roloc_graph_node" output="screen">
    <remap from="/range" to="/idolink_node/anchor/range" />
    <remap from="/scan" to="/scan360" />
    <param name="update_rate" value="5.0"/>
    <param name="update_min_d" value="0.5"/>
    <param name="update_min_a" value="0.1"/>
    <param name="init_x" value="-7.032"/>
    <param name="init_y" value="-10.61"/>
    <param name="init_a" value="1.63"/>
    <param name="height_correction" value="1.8"/>
    <param name="publish_graph" value="-1"/>
    <param name="max_poses" value="50"/>
    <param name="range_weight" value="0.1"/>
    <param name="base_frame_id" value="base_link"/>
    <param name="odom_frame_id" value="odom"/>
    <param name="global_frame_id" value="map"/>
    <param name="transform_tolerance" value="0.2"/>
  </node>
   
</launch>
